app/Repositories/TweetRepository.php

```diff
@@ -32,6 +33,9 @@ public function store(TweetEntity $tweetEntity)
 
     public function find($tweet_id)
     {
+        $tweet = Cache::remember('find_tweet', 60 * 60 * 24, function () use ($tweet_id) {
```

Comment:
What will happen if I tried to `find(1)` and `find(2)`? will it return the same tweet?